sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - We are now targeting the service's v3.1-preview.1 API as the default. If you would like to still use version v3.0 of the service,
 pass in `v3.0` to the kwarg `api_version` when creating your TextAnalyticsClient
 - We have added an API `recognize_pii_entities` which returns entities containing personal information for a batch of documents. Only available for API version v3.1-preview.1 and up.
+- Added `offset` and `length` properties for `CategorizedEntity`, `SentenceSentiment`, and `LinkedEntityMatch`. Tells the user the length
```

Comment:
nit: "Tells the user..." seems out of place since this is written for the user to read.